enterprise/backend/src/metabase_enterprise/sso/integrations/saml.clj

```diff
@@ -137,7 +137,8 @@
     attrs))
 
 (defn- base64-decode [s]
-  (codecs/bytes->str (codec/base64-decode s)))
+  (when (u/base64-string? s)
+    (codecs/bytes->str (codec/base64-decode s))))
```

Comment:
this looks like a duplicate of the saml fix

Reply:
Yeah, bumping the version of Apache commons codecs actually caused this to stop throwing an Exception when you try to base64-decode something that's not valid base 64, which caused a test failure -- I added this check here to fix that. I made the same change in the SAML branch after the fact since I was working on both at the same time and I didn't want merge conflicts. It's not *actually* required there